### PR TITLE
tests: run all tests, not just one spec

### PR DESCRIPTION
**Pull request by nickumia-reisys**
Replaces https://github.com/GSA/inventory-app/pull/369

### CODE REVIEW

**Comment by nickumia-reisys** [comment]
I can't figure out why `dataset.spec.js` is failing, @jbrown-xentity I might (read: will) need your help on Monday. 😖

**Comment by jbrown-xentity** [comment]
Needs https://github.com/GSA/ckanext-usmetadata/pull/56 to be merged for tests to pass, and need to figure out why tests run after turning on docker image for web service and why they fail when using `docker run`...

**Comment by jbrown-xentity** [comment]
This now does a whole lot:

- Merges catalog functionality to inventory (user setup, prerun.py, use .env file, remove development.ini and docker_test.ini)
- Fixes and adds a few tests around resource loading
- Rolls back to ckan2.9.3 (breaking resource load)
- Updates ckanext-usmetadata to avoid breaking changes
- Performs e2e UI tests
- Performs CKAN extension unit tests

File: e2e/cypress/integration/organizations.spec.js [hunk 1]
```diff
@@ -25,6 +25,10 @@ describe('Organization', () => {
     
     it('Edit Organization Description', () => {
         cy.visit('/organization/edit/cypress-test-org')
+
+        // Hide flask debug toolbar
+        cy.get('#flHideToolBarButton').click();
```
Comment (nickumia-reisys) [inline e2e/cypress/integration/organizations.spec.js:30]: Should this be a recurring thing?  Why is this showing up all of a sudden?  Or if it was always there, why is it causing an issue now?

**Review by nickumia-reisys — COMMENTED** [review]
Apart from the oddity mentioned, it looks good to me.  A lot of good improvements that we've talked about before 💯 

And alas, I can't approve this because I'm the author 😆

**Review by jbrown-xentity — APPROVED** [review]

File: e2e/cypress/integration/organizations.spec.js (hunk 1 above)
Comment (jbrown-xentity) [reply e2e/cypress/integration/organizations.spec.js:30]: Now the dev environment is actually being used properly, so it's a thing now. We've handled this on catalog from the beginning...

Comment (nickumia-reisys) [reply e2e/cypress/integration/organizations.spec.js:30]: Got it.  Makes sense (******************nods head******************)

**Pull request merged**